mimic/core.py

```diff
@@ -35,13 +35,16 @@ def __init__(self, clock, apis, domains=()):
             and determining timestamps.
         :type clock: :obj:`twisted.internet.interfaces.IReactorTime`
 
-        :param apis: an iterable of all :obj:`IAPIMock`s that this MimicCore
-            will expose.
+        :param apis: an iterable of all :obj:`IAPIMock` and
+            :obj:`IAPIExternalAPI` mocks that this MimicCore will expose.
 
         :param domains: an iterable of all :obj:`IAPIDomainMock`s that this
             MimicCore will expose.
         """
-        self._uuid_to_api = {}
+        self._uuid_to_api = {
```

Comment:
Ahem. Implied in previous comment:

- replace `_uuid_to_api['internal']` with just `_uuid_to_api`, since this will make for a smaller change and preserve the type of the variable through history better.
- "both of these" for `:ivar:` on `MimicCore` meaning both the `_uuid_to_api` ivar and the `_uuid_to_external` ivar.